season_5/down_in_the_river_to_pray.ly

```diff
@@ -0,0 +1,729 @@
+\version "2.21.6"
+
+\include "../../LilypondTemplates/midi.ly"
+
+Title = "down_in_the_river_to_pray"
+
+\header {
+  title = "Down in the River to Pray"
+  composer = "George H. Allan, Brock Williard"
+  poet = "Slave Songs of the US"
+  tagline = ##f
+}
+
+sopranoOneMusic = \relative c''{
+  \key c \major
+  \numericTimeSignature
+  \time 2/2
+  \tempo 2=55
+  \set Score.markFormatter = #format-mark-circle-numbers
+
+  % lead-in
+  r1
+  r1
+
+  % alto/tenor duet
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
+  r1 r1 r1 r1 r1 r1 r1 r1 r1
```

Comment:
This is optional; but I like putting bar checks in the music.  It catches lots of errors; at least it does for me (wrong duration, skipped a note, forgot a change in time signature, etc.).

Also, for one of the voices I like to add forced line breaks (and stop linebreaks) by adding "\noBreak", and "\break" respectively as needed to force our systems to match the systems in the original music.  I find that it's much easier to make a visual inspection of notes that way.